src/chainlit/frontend/src/components/organisms/chat/inputBox/input.tsx

```diff
@@ -41,7 +42,7 @@ const Input = ({ onSubmit, onReply }: Props) => {
   const [isComposing, setIsComposing] = useState(false);
 
   const socketOk = session?.socket && !session?.error;
-  const disabled = !socketOk || loading || askUser?.spec.type === 'file';
+  const disabled = !socketOk || loading;
```

Comment:
why remove the `askUser?.spec.type === 'file'` condition? If the server is waiting for a file the input box should be disabled